sdk/core/Azure.Core/tests/ResponseClassifierTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Pipeline;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests
+{
+    public class ResponseClassifierTests
+    {
+        [Theory]
+        [TestCase(null, false, null)]
```

Comment:
I would suggest including `application/json` as a case for a text response type.